src/prompt-input/__tests__/prompt-input.test.tsx

```diff
@@ -104,18 +110,44 @@ describe('action button', () => {
   });
 
   test('present when added', () => {
-    const { wrapper } = renderPromptInput({ value: '', actionButtonIconName: 'send' });
+    const { wrapper } = renderPromptInput({
+      value: '',
+      actionButtonIconName: 'send',
+    });
     expect(wrapper.findActionButton().getElement()).toBeInTheDocument();
   });
 
+  test('should render action button inside secondary actions container when secondary actions are present', () => {
+    const { wrapper } = renderPromptInput({
+      value: '',
+      minRows: 4,
+      secondaryActions: 'secondary actions',
+      actionButtonIconName: 'send',
+    });
+
+    const secondaryActionsContainer = wrapper.findSecondaryActions()!.getElement();
+    const actionButton = within(secondaryActionsContainer).getByRole('button');
+
+    expect(actionButton).toBeInTheDocument();
+  });
+
   test('disabled when in disabled state', () => {
-    const { wrapper } = renderPromptInput({ value: '', actionButtonIconName: 'send', disabled: true });
+    const { wrapper } = renderPromptInput({
+      value: '',
+      actionButtonIconName: 'send',
+      disabled: true,
+    });
     expect(wrapper.findActionButton().getElement()).toHaveAttribute('disabled');
   });
 
-  test('disabled when in read-only state', () => {
-    const { wrapper } = renderPromptInput({ value: '', actionButtonIconName: 'send', readOnly: true });
-    expect(wrapper.findActionButton().getElement()).toHaveAttribute('disabled');
+  test('focusable when in read-only state', () => {
```

Comment:
This test does not seem to actually test that the element is focusable (which as far as I know cannot be done in a unit test using JSDOM, but has to be done in a browser, i.e. an integ test)

Reply:
Good point. I should update the name of this test to be what it should be testing for, which is the addition of the disabled aria tag.